sys-cluster/galera/files/galera-26.4.10-gcomm-test-check-fix.patch

```diff
@@ -0,0 +1,12 @@
+# Patches from Alpine linux:
+# https://git.alpinelinux.org/aports/tree/testing/galera/fix_gcomm-test-check_evs2.patch
+--- a/gcomm/test/check_evs2.cpp
++++ b/gcomm/test/check_evs2.cpp
+@@ -2583,6 +2583,7 @@ START_TEST(test_representative_incarnation_change)
+     gu::datetime::SimClock::inc_time(300 * gu::datetime::MSec);
+     evs_from_dummy(dn[1])->handle_timers();
+     evs_from_dummy(dn[2])->handle_timers();
++    prop.propagate_until_empty();
```

Comment:
Everything looks okay apart from this one. It stands out as a bit odd.

Has this been sent upstream somewhere?

Reply:
Uhm no. It was there alpine Linux perhaps I can wrap it around ifdef

Reply:
Ping.